amundsen_application/static/js/config/config-default.ts

```diff
@@ -38,7 +38,7 @@ const configDefault: AppConfig = {
     notificationsEnabled: false,
   },
   announcements: {
-    enabled: true,
+    enabled: false,
```

Comment:
fyi our internal version will need to be updated to `true` when we update the submodule.